feat(dashboardsv2): revisions with more screenshots (#3200)
getsentry/sentry-docs: src/docs/product/dashboards/index.mdx

</Note>

> Dashboards will enable you to get a broad overview of your application’s health. You’ll be able to navigate through error and performance data across multiple projects and zoom in on any time series visualizations you may want to investigate. All widgets in the same view will update synchronously to reflect the new date range in the Global Selection Header.

Reviewer: Rewording for brevity. Good rule of thumb to replace "will [verb]" with one verb wherever you can unless the thing you're talking about is truly going to only happen in the future. 

The last sentence talks about widgets without having introduced them, so I edited this so that widgets get introduced. It's closer to the original text but without mentioning building custom dashboards. Also moved some text to after the image.

```suggestion
Dashboards provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/). 
```

Author: Changed

> ![Widgets visualizing events, errors by country, affected users, and handled vs unhandled issues.](dashboard_default.png)

Reviewer: ```suggestion
![Widgets visualizing events, errors by country, affected users, and handled vs unhandled issues.](dashboard_default.png)

All widgets in the same view reflect the date range indicated in the Global Selection Header and update synchronously if you update that date range. You can also zoom in on any time series visualizations you may want to investigate, and all of the widgets reflect the time period that you've zoomed in on.
```

Author: Changed

> ## Default View
> The view you’ll start out with will contain the following widgets:

Reviewer: I'm not sure why we wouldn't refer to this as the Default Dashboard. I don't know if it's helpful to start using the term "view" and "dashboard" interchangeably when we can just use one that refers back to the feature. I've made my suggestion below with that in mind.

```suggestion
## Default Dashboard
The default dashboard includes the following widgets:
```

Author: Changed

> - Number of Errors - Shows the total numbers of errors you have.
> - Number of Issues - Shows the total number of issues you have.
> - All Events - Shows all errors, transactions, CSP, and more.
> - Affected Users - Compares Known Users v. Unknown Users.
> - Handled v. Unhandled - Compares Handled Errors v. Unhandled Errors.
> - Errors by Country - Shows a world map of error frequency.
> - Errors by Browser - Compares number of errors by browser name.

Reviewer: Minor edits. Should spell out what CSP means though.

```suggestion
- Number of Errors: Shows the total numbers of errors you have.
- Number of Issues: Shows the total number of issues you have.
- All Events: Shows all errors, transactions, CSP, and more.
- Affected Users: Compares Known Users to Unknown Users.
- Handled v. Unhandled: Compares Handled Errors to Unhandled Errors.
- Errors by Country: Shows a world map of error frequency.
- Errors by Browser: Compares the number of errors by browser name.
```

Author: Changed
